components/domains/registerCheckboxes.tsx

```diff
@@ -58,15 +62,22 @@ const RegisterCheckboxes: FunctionComponent<RegisterCheckboxes> = ({
           </p>
         </div>
         {!isArOnforced ? (
-          <div
-            className="flex items-center justify-left text-xs cursor-pointer"
-            onClick={onChangeRenewalBox}
+          <InputHelper
+            helperText={`Enabling a subscription permits Starknet ID to renew your domain automatically every year for you! This approval gives us only the possibility to renew your domain once per year maximum (${gweiToEth(
+              ethRegistrationPrice
```

Comment:
Nope! Because here if someone registers for 2 years he'll have 0.018 instead of 0.009. We need only the price for one year to be shown here !

Reply:
Ah yes my bad thanks!